### PR TITLE
Fixes some things with gimmick positions

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

Adds gimmick position overhead chat colours.
Adds gimmick position colours to the actual chat.
Fixes them having no message on the selection menu.
image
^ that

## Why It's Good For The Game

Bug fixes.

## Changelog
:cl:
fix: Gimmick positions now have overhead chat colours.
fix: Gimmick positions now have chat colours.
fix: Gimmick positions now have a title in the job selection menu.
/:cl:

### CODE REVIEW

**Review by ike709 — DISMISSED** [review]

**Comment by PowerfulBacon** [comment]
hold on

**Comment by PowerfulBacon** [comment]
Ready

**Review by ike709 — DISMISSED** [review]

File: code/modules/jobs/jobs.dm [hunk 1]
```diff
@@ -78,7 +78,8 @@ GLOBAL_LIST_INIT(exp_jobsmap, list(
 	EXP_TYPE_SUPPLY = list("titles" = supply_positions),
 	EXP_TYPE_SECURITY = list("titles" = security_positions),
 	EXP_TYPE_SILICON = list("titles" = list("AI","Cyborg")),
-	EXP_TYPE_SERVICE = list("titles" = civilian_positions | gimmick_positions),
+	EXP_TYPE_SERVICE = list("titles" = civilian_positions),
```
Comment (PowerfulBacon) [inline code/modules/jobs/jobs.dm]:
```suggestion
	EXP_TYPE_SERVICE = list("titles" = civilian_positions | gimmick_positions),
```

**Review by park66665 — APPROVED** [review]
All clear. This PR is good to go.

**Pull request merged**